src/createNumbersGenerator.js

```diff
@@ -26,7 +26,25 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let count = 0;
+  return () => {
+    if (numbers === undefined) {
+      return undefined;
+    } else {
```

Comment:
no need for this else
```suggestion
    }
```